set.mm

```diff
@@ -34332,7 +34332,9 @@ something like (wi (wceq (cv vx) (cv vy)) wph) ) into just (wcdeq vx vy
        "collection of all sets which are not members of themselves" is a set.
        However, here we prove ` { x | x e/ x } e/ _V ` .  This contradiction
        was discovered by Russell in 1901 (published in 1903), invalidating the
-       Comprehension Axiom and leading to the collapse of Frege's system.
+       Comprehension Axiom and leading to the collapse of Frege's system, which
+       Frege acknowledged in the second edition of his _Grundgesetze der
+       Arithmetik_.
```

Comment:
We don't want to be an encyclopedia in the history of mathematics, but I think that fact is important enough to be mentioned (and is an example of intellectual integrity from Frege).